### PR TITLE
Add support for creating gpdb7 debian packages

### PR DESCRIPTION
**Pull request by bhanu-atturu**
Make change in build and test scripts for gpdb7

Authored-by: Bhanu Kiran Atturu <batturu@vmware.com>

### CODE REVIEW

File: ci/concourse/scripts/build-gpdb-deb.bash [hunk 1]
```diff
@@ -46,7 +46,12 @@ function build_deb() {
 	cat <<EOF >"${__package_name}/${DEB_DIR}/prerm"
 #!/bin/sh
 set -e
-dpkg -L "greenplum-db-${gpdb_major_version}" | grep '\.py$' | while read file; do rm -f "\${file}"[co] >/dev/null; done
```
Comment (BaiShaoqi) [inline ci/concourse/scripts/build-gpdb-deb.bash:49]: why this change only goes to greenplum-db-7 , shouldn't it be also goes to greenplum-db-6 ?

File: ci/concourse/scripts/build-gpdb-deb.bash [hunk 2]
```diff
@@ -46,7 +46,12 @@ function build_deb() {
...
+if [ "${gpdb_major_version}" = "7" ]; then
+	cd ${GPDB_PREFIX}/${GPDB_NAME}-${GPDB_VERSION}
+	find . | grep __pycache__ | xargs rm -rf
```
Comment (BaiShaoqi) [inline ci/concourse/scripts/build-gpdb-deb.bash:51]: Curious why it is not possible to apply the old command `dpkg` to greenplum-db-7 ? although find may also work.

File: ci/concourse/scripts/test-functionality-deb.bash [hunk 3]
```diff
@@ -7,12 +7,14 @@ if [[ ${PLATFORM} != "ubuntu"* ]]; then
...
+mv gpdb_deb_installer/*.deb greenplum-database-release/gpdb-deb-test/gpdb_deb_installer/greenplum-db-ubuntu18.04-amd64.deb
 pushd greenplum-database-release/gpdb-deb-test
 godog features/gpdb-deb.feature
 popd
 
-mv gpdb_deb_ppa_installer/*.deb greenplum-database-release/gpdb-deb-test/gpdb_deb_installer/greenplum-db-6-ubuntu18.04-amd64.deb
-pushd greenplum-database-release/gpdb-deb-test
-godog features/gpdb-ppa.feature
-popd
+if [[ ${PPA} == "true" ]]; then
```
Comment (BaiShaoqi) [inline ci/concourse/scripts/test-functionality-deb.bash:15]: minor tip, we can skip PPA, since you can check gpdb_deb_ppa_installer is not empty as same logic

File: ci/concourse/scripts/build-gpdb-deb.bash (hunk 2 above)
Comment (BaiShaoqi) [reply ci/concourse/scripts/build-gpdb-deb.bash:51]: So python3 will generate .pyc files under __pycache__ , if that is the case, I think we can still use the old dpkg for greenplum-db-7,  to me, dpgk is still a better way to figure out if a file belongs to deb package.

**Review by BaiShaoqi — APPROVED** [review]
overall, looks good to me

File: ci/concourse/scripts/test-functionality-deb.bash (hunk 3 above)
Comment (bhanu-atturu) [reply ci/concourse/scripts/test-functionality-deb.bash]: thanks, updated.

File: ci/concourse/scripts/build-gpdb-deb.bash (hunk 2 above)
Comment (bhanu-atturu) [reply ci/concourse/scripts/build-gpdb-deb.bash:51]: if we leave an empty pycache dir, then during uninstall, /usr/local/greenplum-db-${VERSION} will not be removed completely.
